code/game/objects/objs.dm

```diff
@@ -8,6 +8,11 @@
 	var/damtype = BRUTE
 	var/force = 0
 
+	/// How good a given object is at causing wounds on carbons. Higher values equal better shots at creating serious wounds.
+	var/wound_bonus = 0
+	/// If this attacks a human with no wound armor on the affected body part, add this to the wound mod. Some attacks may be significantly worse at wounding if there's even a slight layer of armor to absorb some of it vs bare flesh
+	var/bare_wound_bonus = 0

```

Comment:
This is not very generic behavior.

Reply:
[Continued from here](https://github.com/tgstation/tgstation/pull/50558#discussion_r413572687)

Unsure what specifically you mean by non-generic behavior.

If you mean having a special bonus against bare skin as well as a general wound_bonus, basic clothes provide you with a little bit of wound protection, but sharp and burning weapons can shred this protection with repeated hits. This way, one or two scalpel stabs or stray laser blasts won't immediately cripple you, but someone attacking you repeatedly with those will be a lot deadlier after the 2nd-3rd hit.

If you mean why it's only for humans specifically, humans are the only mobs (other than corgis and drones I guess) that can actually have armor, so they're the only ones we care about

Reply:
@Ryll-Ryll can't monkeys wear helmets?